src/pages/chooseValidity.tsx

```diff
@@ -78,19 +112,27 @@ export const getServerSideProps = (ctx: NextPageContextWithSession): { props: Va
         throw new Error('Failed to retrieve passenger type session info for choose validity page.');
     }
 
-    let validity;
+    let amount;
+    let timePeriodType;
```

Comment:
If we assign both of these variables as an empty string here, we won't need to default them in the returned props (lines 133 and 135)

Reply:
Do we gain anything or lose anything either way?